google/cloud/storage/internal/grpc/monitoring_project_test.cc

```diff
@@ -30,6 +33,26 @@ namespace {
 using ::google::cloud::testing_util::ScopedEnvironment;
 using ::testing::Optional;
 
+TEST(MonitoringProject, Resource) {
+  EXPECT_EQ(MonitoringProject(opentelemetry::sdk::resource::Resource::Create(
+                {{"cloud.region", "unknown"}})),
+            absl::optional<Project>());
```

Comment:
nit: here and below, I think we more commonly say just: `absl::nullopt`

Reply:
Done.